code/wdb/datastripe.q

```diff
@@ -2,73 +2,100 @@
 
 segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
 
-td:hsym `$getenv`KDBTAIL
+td:hsym `$"/"sv (getenv`KDBTAIL;string .z.d)
```

Comment:
I think the latest structure we are going with is 
data/taildir/IDB1/2022.06.09
                  /IDB2/2022.06.10
                  
I think in other parts of torq (e,g, wdb.q) they use pt or the current partition variable as the partition to look at, instead of using .z.d.  Then at EOD this value is increased by one once the processes are fine to move onto the next day/partition. Just thinking that at the EOD there might be a period where we want the access tables from yesterday whilst the write down to HDB is taking place.

Reply:
I'm not sure where I was going with this either now